next.config.js

```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  images: {
+    domains: ["images.unsplash.com", "nftstorage.link"],
```

Comment:
what are we using `unsplash` for? 

any reason to not store those photos on nft storage?

Reply:
not using it for anything